README.md

```diff
@@ -11,27 +11,32 @@ export KEY_NAME=mozilla_vitillo
 aws emr create-cluster \
   --region us-west-2 \
   --name SparkCluster \
-  --ami-version 3.3.2 \
   --instance-type c3.4xlarge \
-  --instance-count 5  \
+  --instance-count 1 \
   --service-role EMR_DefaultRole \
-  --ec2-attributes KeyName=${KEY_NAME},InstanceProfile=$SPARK_PROFILE \
-  --bootstrap-actions Path=s3://support.elasticmapreduce/spark/install-spark,Args=["-v","1.3.1.e"] Path=s3://${SPARK_BUCKET}/telemetry.sh Path=s3://elasticmapreduce/bootstrap-actions/configure-hadoop,Args=["-y","yarn.nodemanager.vmem-check-enabled=false","-y","yarn.nodemanager.pmem-check-enabled=false"]
+  --ec2-attributes KeyName=${KEY_NAME},InstanceProfile=${SPARK_PROFILE} \
+  --release-label emr-4.1.0 \
```

Comment:
We should switch to 4.2.0, since that came out recently and should be compatible (= spark 1.5.2).